jenkins/scripts/integration_test.sh

```diff
@@ -49,7 +49,7 @@ VM_TIMELABEL="${VM_TIMELABEL:-$(date '+%Y%m%d%H%M%S')}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME:-ci-test-vm-${VM_TIMELABEL}}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 
-if [[ "${TESTS_FOR}" == "feature_tests"* ]]
+if [[ "${TESTS_FOR}" == "feature_tests"* || ${"TEST_FOR"} == "e2e_tests*" ]]
```

Comment:
should not this be:
```suggestion
if [[ "${TESTS_FOR}" == "feature_tests"* || ${"TEST_FOR"} == "e2e_tests"* ]]
```
?

The meaning of "*" wildcard in case of `feature_tests` was if it matches substring(feature_tests) of string(i.e feature_tests_centos), it will meet the requirement, meaning if we will have `e2e_tests` for more than one flavor we could do the same here.

Reply:
Yes, of course :sweat_smile:

Reply:
Updated